community/kernel/src/main/java/org/neo4j/kernel/impl/query/Neo4jTransactionalContext.java

```diff
@@ -245,12 +234,6 @@ public Statement statement()
     }
 
     @Override
-    public void check()
```

Comment:
hm why we are removing this? That thing was recently introduced (that's why it's not used) for cypher land to be able to check if current transaction is still valid while doing time consuming operations (lets imaging huge sorting, etc.)